<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="smallDecimalPoint">
        <bool>true</bool>
       </property>
       <property name="digitCount">
        <number>4</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_request">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Request</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Hold">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Hold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_hv">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>HV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_connect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
